batch-models/src/main/scala/org/ekstep/analytics/exhaust/OnDemandDruidExhaustJob.scala

```diff
@@ -185,6 +194,7 @@ object OnDemandDruidExhaustJob extends BaseReportsJob with Serializable with IJo
             val reportConfig = JSONUtils.deserialize[ReportConfig](JSONUtils.serialize(updatedReportConf))
             val storageConfig = getStorageConfig(config, AppConf.getConfig("collection.exhaust.store.prefix"))
             JobLogger.log("Total Requests are ", Some(Map("jobId" -> jobId, "totalRequests" -> requests.length)), INFO)
+            println(JSONUtils.serialize(reportConfig))
```

Comment:
Remove the println statements

Reply:
ok @sowmya-dixit . Will remove this

Reply:
@sowmya-dixit its done. Please check and merge the PR